src/Client/Collections.php

```diff
@@ -53,6 +54,19 @@ public function forSubject(string ...$subjectId) : self
         return $clone;
     }
 
+    public function containing(string ...$items) : self
```

Comment:
Annoying to pass strings here, maybe have value objects for content identifiers?